src/GitVersion.Core/MergeMessage.cs

```diff
@@ -96,13 +96,10 @@ private ReferenceName GetMergedBranchName(string mergedBranch)
         mergeCommit.NotNull();
         configuration.NotNull();
 
-        mergeMessage = null;
+        mergeMessage = new(mergeCommit.Message, configuration);
```

Comment:
`mergeMessage` should not be set if the return value equals `false`.

Reply:
Understood, I've changed it.